crates/connector_configs/src/connector.rs

```diff
@@ -80,6 +80,17 @@ pub enum KlarnaEndpoint {
     Oceania,
 }
 
+#[serde_with::skip_serializing_none]
+#[derive(Debug, Deserialize, serde::Serialize, Clone)]
+pub struct ConfigMerchantAdditionalDetails {
+    pub open_banking_recipient_data: Option<MetaDataInupt>,
+    pub account_data: Option<MetaDataInupt>,
+    pub iban: Option<Vec<MetaDataInupt>>,
+    pub bacs: Option<Vec<MetaDataInupt>>,
```

Comment:
```suggestion
    #[serde(default)]
    pub iban: Vec<MetaDataInupt>,
    #[serde(default)]
    pub bacs: Vec<MetaDataInupt>,
```

does `None` & `Some([])` behave differently in the application ? if not we can use the above